apps/financial-aid/backend/src/app/modules/application/emailTemplates/applicantEmailTemplate.ts

```diff
@@ -127,7 +127,11 @@ export const ApplicantEmailTemplate = (emailData: ApplicantEmailData) => {
                          <tr> 
                           <td align="center" style="padding:0;Margin:0;padding-bottom:30px"><span class="es-button-border" style="border-style:solid;border-color:#2CB543;background:#2CB543;border-width:0px 0px 2px 0px;display:inline-block;border-radius:30px;width:auto;border-top-left-radius:8px;border-top-right-radius:8px;border-bottom-right-radius:8px;border-bottom-left-radius:8px;background-color:#0061ff;border-bottom-width:0px"><a href="${
                             emailData.applicationLink
-                          }" class="es-button es-button-1" target="_blank" style="mso-style-priority:100 !important;text-decoration:none;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;color:#FFFFFF;font-size:18px;border-style:solid;border-color:#0061ff;border-width:18px 24px;display:inline-block;background:#31CB4B;border-radius:30px;font-family:'IBM Plex Sans', 'open sans', 'helvetica neue', helvetica, arial, sans-serif;font-weight:normal;font-style:normal;line-height:22px;width:auto;text-align:center;background-color:#0061ff;border-top-left-radius:8px;border-top-right-radius:8px;border-bottom-right-radius:8px;border-bottom-left-radius:8px">Bæta við gögnum</a></span></td> 
+                          }" class="es-button es-button-1" target="_blank" style="mso-style-priority:100 !important;text-decoration:none;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;color:#FFFFFF;font-size:18px;border-style:solid;border-color:#0061ff;border-width:18px 24px;display:inline-block;background:#31CB4B;border-radius:30px;font-family:'IBM Plex Sans', 'open sans', 'helvetica neue', helvetica, arial, sans-serif;font-weight:normal;font-style:normal;line-height:22px;width:auto;text-align:center;background-color:#0061ff;border-top-left-radius:8px;border-top-right-radius:8px;border-bottom-right-radius:8px;border-bottom-left-radius:8px">${
+    emailData.applicationLinkText
+      ? emailData.applicationLinkText
+      : 'Bæta við gögnum'
```

Comment:
This is kind of weird default message, shouldn't we rather have `applicationLinkText` on more emailDatas and show that and if it needs to be nullable then we can just have 'Opna stöðusíðu' ?